source/_patterns/04-pages/article--research-aside.mustache

```diff
@@ -0,0 +1,91 @@
+<div class="global-wrapper" data-item-type="research-article">
```

Comment:
After a call with Nathan, I created a new page for aricle research pages with aside element, the original page for article research was reverted